### ISSUE TITLE
Torch thrown on torch unplacable blocks (like leaves) disappear  

### ISSUE THREAD

**Issue by Ch1216** [DanikingMC/ThrowableTorch#9]
It should drop a torch like when you try to put a torch on the underside of a block or maybe just the whole item/torch with whatever you threw it with

**Comment by DanikingMC** [comment]
Yes, it should drop the torch. I'll need to change the logic of throwable torches at the moment they hit those blocks. Thanks for supporting
